defaults/stylesheet.css

```
@@ -0,0 +1,89 @@
+.wf-history-node + .wf-history-node, .wf-history-node .wf-history-node { margin-top: 8px; }
+.wf-history-node--history { background: #f8faf7; }
+.wf-history-node--state { background: #eef3ef; }
+.wf-history-node--manifestation { background: #e7eee9; }
+.wf-history-node--file { background: #dfe8e2; }
+.wf-history-node-header { display: flex; flex-wrap: wrap; gap: 8px; align-items: baseline; }
+.wf-history-node > summary.wf-history-node-header { display: list-item; padding: 0; }
+.wf-history-class { color: #687167; font-style: italic; font-size: 0.85rem; }
+.wf-history-file-iri { font-size: 0.82rem; overflow-wrap: anywhere; }
+.wf-history-gap { display: flex; align-items: center; justify-content: center; min-height: 28px; color: #687167; font-size: 1.15rem; line-height: 1; }
+.wf-source-meta { display: flex; flex-wrap: wrap; gap: 10px; padding: 0 14px 12px; color: #596259; font-size: 0.88rem; }
+pre { margin: 0; width: 100%; max-width: 100%; max-height: 64vh; overflow: auto; border-top: 1px solid #d7dcd4; background: #0d1117; color: #e6edf3; padding: 16px; font-size: 0.86rem; line-height: 1.55; tab-size: 2; white-space: pre-wrap; overflow-wrap: anywhere; word-break: break-word; }
```

Stylelint (17.12.0)

[error] Deprecated keyword "break-word" for property "word-break" (declaration-property-value-keyword-no-deprecated)

(declaration-property-value-keyword-no-deprecated)